src/Esfa.Vacancy.Register.Api/Mappings/VacancyMappings.cs

```diff
@@ -1,33 +1,124 @@
-﻿using AutoMapper;
+﻿using System.Globalization;
```

Comment:
We can take all this mapping logic and create a actual mapping class by hand. It has been decided that Automapper doesn't really benefit situations like these.